roles/addons/diskless/readme.rst

```diff
@@ -122,16 +122,38 @@ Select the kernel you want to use wit this image from the list of available kern
  
 Choose a standard image, and give a size.
 
+Notes:
+
+* The size recommended for the Standard or Custom is 5G
+* The size of the final image booted and loaded in RAM is approximately : 1.2GB for Standard image, 250MB for Small image, 130MB for Minimall image and 1.2GB for Custom image
```

Comment:
Can you make a list in the same order than the option in the CLI?

 - 1.2GB for Standard...
 - 250 MB for Small
 - 130 MB for Minimal (typo: minimall)

No size for Custom since it is impossible to predict the final size.

Reply:
ok